tests/conftest.py

```
@@ -1,15 +1,25 @@
 import os
 import shutil
 import sys
+import tempfile
 from pathlib import Path
+from unittest.mock import MagicMock, patch
```

Ruff (0.8.2)

`unittest.mock.MagicMock` imported but unused

Remove unused import

(F401)